.github/workflows/ci-pr.yml

```diff
@@ -5,18 +5,11 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
-    name: CI PR Build
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-java@v2
-        with:
-          distribution: adopt
-          java-version: 8
-          cache: maven
-      - name: Build kafka-samples
-        run: ./mvnw -B package
-        working-directory: kafka-samples
-      - name: Build multio-samples/ticktock
-        run: ./mvnw -B package
-        working-directory: multio-samples/ticktock
+    name: Call Build Workflow
+    uses: spring-cloud/spring-cloud-dataflow-samples/.github/workflows/ci.yml@main
+    with:
+      maven-build-only: true
```

Comment:
This let's use completely re-use the main CI workflow excluding the Artifactory publish and Docker push for all of the sample apps (basically, just the build and test step(s)).